packages/now-node/src/index.ts

```diff
@@ -281,6 +281,19 @@ async function compile(
   };
 }
 
+function getAWSLambdaHandler(entrypoint: string, config: Config) {
+  if (config.awsLambdaHandler) {
+    return config.awsLambdaHandler as string;
+  }
+
+  if (process.env.NODEJS_AWS_HANDLER_NAME) {
+    const { name } = parsePath(entrypoint);
+    return `${name}.${process.env.NODEJS_AWS_HANDLER_NAME}`;
```

Comment:
The path might be relevant still as seen in the `25-aws-api` test fixture:

https://github.com/zeit/now/blob/057f9f7045f31471a0cc1f65d86b45129cd5b01f/packages/now-node/test/fixtures/25-aws-api/now.json#L12